spec/controllers/spree/admin/reports_controller_spec.rb

```diff
@@ -88,6 +88,18 @@
         assigns(:search).result.should_not include(orderA2)
         assigns(:search).result.should_not include(orderB2)
       end
+
+      it 'returns accessible line items in json' do
+        spree_get :orders_and_distributors, format: :json
+
+        json = JSON.parse(response.body)
+
+        json['line_items'].size.should == 4
+        json['orders'].size.should == 2
+        json['products'].size.should == 2
+        json['distributors'].size.should == 1
+        json['variants'].size.should == 2
```

Comment:
I agree. This since is deprecated since early 2014, so the sooner we move away from it, the better.